build.sbt

```diff
@@ -1,7 +1,7 @@
 lazy val slf4s = (project in file(".")).settings(
   organization := "org.slf4s",
   name := "slf4s-api",
-  scalaVersion := "2.11.6",
+  scalaVersion := "2.11.7",
   scalacOptions := Seq("-unchecked", "-deprecation", "-language:experimental.macros"),
   version := "1.7.12",
```

Comment:
I've incremented this in https://github.com/mattroberts297/slf4s/commit/6cd283977bb65150daa30664f4c086def54aaa5b. I've changed the `slf4j-api` dependency version to `version.value` in the same way you changed the `scala-reflect` version.